docs/limitations.md

```diff
@@ -1,9 +1,43 @@
 # Clear Containers known differences and limitations
 
+* [Pending items](#pending-items)
+    * [Docker swarm support](#docker-swarm-support)
+    * [Networking](#networking)
+        * [Adding networks dynamically](#adding-networks-dynamically)
+    * [Resource management](#resource-management)
+        * [`docker run --cpus=`](#`docker-run---cpus=`)
+        * [`docker run --kernel-memory=`](#`docker-run---kernel-memory=`)
+        * [shm](#shm)
+        * [cgroup constraints](#cgroup-constraints)
+        * [Capabilities](#capabilities)
+        * [sysctl](#sysctl)
+        * [tmpfs](#tmpfs)
+    * [Other](#other)
+        * [checkpoint and restore](#checkpoint and restore)
+        * [`docker stats`](#`docker-stats`)
+    * [runtime commands](#runtime-commands)
+        * [`ps` command](#`ps` command)
+        * [`events` command](#`events`-command)
+        * [`update` command](#`update`-command)
+  * [Architectural limitations](#architectural-limitations)
+      * [Networking](#networking)
+        * [Support for joining an existing VM network](#support-for-joining-an-existing-vm network)
+        * [`docker --net=host`](#`docker---net=host`)
+        * [`docker run --link`](#`docker-run---link`)
+    * [Host resource sharing](#host-resource-sharing)
+        * [`docker --device`](#`docker---device`)
+        * [`docker -v /dev/...`](#`docker--v-/dev/...`)
+        * [`docker run --privileged`](#`docker-run---privileged`)
+    * [Other](#other)
+        * [Annotations](#annotations)
+    * [runtime commands](#runtime-commands)
+        * [`init` command](#`init`-command)
+        * [`spec` command](#`spec`-command)
```

Comment:
The links with backticks or spaces included in the second part do not work. Only hyphens are supported.  
Lines: 8, 9, 16, 17, 19, 20, 21, 24, 25, 26, 28, 29, 30, 34, 35.